src/components/Paragraph/README.md

```diff
@@ -1,61 +1,105 @@
-	vulputate et. Ut fermentum venenatis auctor. Mauris augue augue, hendrerit
-	quis risus eget, imperdiet feugiat odio. Nullam porta sollicitudin ipsum eu
-	varius. Nam hendrerit eleifend sagittis. Maecenas in sodales libero, nec
-	ultrices ante. Ut et sem nibh. Vivamus fringilla pharetra justo aliquet
-	pharetra.
+	augue gravida hendrerit.
+</Paragraph>
+
+<Paragraph>
+	Vestibulum ornare eget lacus cursus rhoncus. Cras nec
+	lectus in sem viverra volutpat.
 </Paragraph>
 ```
 
-### Medium Paragraph
+## Usage
+
+Use a `Paragraph` for any body copy comprising of one or more full sentences.
+
+Don't use a `Paragraph` for:
+- A single text link or call to action—use a [Button](../#/Function/Components/Button) instead.
+- A heading or subheading—use a `Heading`, `Subtitle`, or `Page Title`.
+- A list of related, equally weighted items—use a `List` instead.
+
+## Appearance
+
+- Line lengths should be between 55-70 characters.
+- Text colour should contrast with the background.
+- Use the large and small variants sparingly when hierarchy is required. Most `Paragraphs` should use the default size.
+
+
+## Variations
+
+Use a **large paragraph** when you want to emphasise an important passage of text.
 
 ```jsx
-<Paragraph>
+<Paragraph large>
 	Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut bibendum hendrerit
 	augue gravida hendrerit. Vestibulum ornare eget lacus cursus rhoncus. Cras nec
-	lectus in sem viverra volutpat. Fusce placerat nunc eu libero egestas, sit
-	amet condimentum turpis ornare. Sed hendrerit metus elit, ac sollicitudin enim
-	vulputate et. Ut fermentum venenatis auctor. Mauris augue augue, hendrerit
-	quis risus eget, imperdiet feugiat odio. Nullam porta sollicitudin ipsum eu
-	varius. Nam hendrerit eleifend sagittis. Maecenas in sodales libero, nec
-	ultrices ante. Ut et sem nibh. Vivamus fringilla pharetra justo aliquet
-	pharetra.
+	lectus in sem viverra volutpat.
 </Paragraph>
 ```
 
-### Small (Fine Print) Paragraph
+Use a **small paragraph** for fine print, or details that are secondary to the main content.
 
 ```jsx
-<Paragraph size="small">
+<Paragraph small>
 	Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut bibendum hendrerit
 	augue gravida hendrerit. Vestibulum ornare eget lacus cursus rhoncus. Cras nec
-	lectus in sem viverra volutpat. Fusce placerat nunc eu libero egestas, sit
-	amet condimentum turpis ornare. Sed hendrerit metus elit, ac sollicitudin enim
-	vulputate et. Ut fermentum venenatis auctor. Mauris augue augue, hendrerit
-	quis risus eget, imperdiet feugiat odio. Nullam porta sollicitudin ipsum eu
-	varius. Nam hendrerit eleifend sagittis. Maecenas in sodales libero, nec
-	ultrices ante. Ut et sem nibh. Vivamus fringilla pharetra justo aliquet
-	pharetra.
+	lectus in sem viverra volutpat.
 </Paragraph>
 ```
 
-## Colors
+Use an **inverse paragraph** if you need to put a paragraph on top of a dark background.
+
+```jsx
+<Paragraph inverse>
```

Comment:
`inverted` is a better prop name, I feel like. Thoughts?

Reply:
"Inverse" means opposite; "inverted" means upside-down!